File: attachment-microservice/src/main/scala/data/repository/AttachmentRepository.scala
package data.repository

import data.entity.Attachment
import org.springframework.data.jpa.repository.{JpaRepository, JpaSpecificationExecutor}
import org.springframework.stereotype.Repository

/**
  * attachment repository
  *
  * @author 刘佳兴
  * @version V1.0
  */
@Repository
trait AttachmentRepository extends JpaRepository[Attachment, java.lang.Long] with JpaSpecificationExecutor[Attachment]

File: attachment-microservice/src/main/java/data/repository/AttachmentRepository.java
package data.repository;

import data.entity.Attachment;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.data.jpa.repository.Query;
import org.springframework.stereotype.Repository;

/**
 * @author 刘佳兴
 */
@Repository
public interface AttachmentRepository extends JpaRepository<Attachment, Long>, JpaSpecificationExecutor<Attachment> {

}
